.github/workflows/deployment.yml

```diff
@@ -56,9 +56,9 @@ jobs:
         id: gitversion
         uses: gittools/actions/gitversion/execute@v0.9.4
       - name: Set up JDK
-        uses: actions/setup-java@v1
+        uses: joschi/setup-jdk@v2
         with:
-          java-version: 15
```

Comment:
Why is the space character deleted? We should keep two spaces consistently; otherwise, we could run into YAML issues.

Reply:
Odd. I modified this in the github workflow. Very odd.